publications-2022.bib

```diff
@@ -661,4 +661,14 @@ @Article{2022:zingaro.fumagalli.ea:geometric
   doi     = {10.3934/dcdss.2022052}
 }
 
+@MastersThesis{2022:Holm:cutfem,
+  author  = {Holm, Sigmund Eggen},
+  title   = {Cut finite element methods for partial differential equations on moving domains},
+  school  = {Norwegian University of Science and Technology},
+  year    = 2022,
+  month   = aug,
+  note    = {Masters's Thesis},
+  url     = {https://ntnuopen.ntnu.no/ntnu-xmlui/handle/11250/3013356}
```

Comment:
```suggestion
  url     = {https://hdl.handle.net/11250/3013356}
```
`@MastersThesis` as bibentry is sufficient, there is no need for the note.
Using `handle.net` is a safer way to provide a link (similar to doi).